### DISCUSSION TITLE
Gov000905@gmail.com

### DISCUSSION THREAD

**Discussion by Gov000905** [iBotPeaches/Apktool#3447] [Q&A]
# Gov000905-gmail.com
on HONOR NTH-NX9 (NTH-NX9) HONOR, Android API 33 Application version: 1.0.11 Build (11)}

**Comment by Gov000905** [comment]
connect:mac:BE:DC:03:90:6C:98] androidx.work.impl.background.systemjob.SystemJobService androidx.work.impl.foreground.SystemForegroundService androidx.work.multiprocess.RemoteWorkManagerService com.franco.kernel.quicktiles.HbmQuickTile com.franco.kernel.quicktiles.Klapse com.franco.kernel.quicktiles.NightShiftTile com.franco.kernel.quicktiles.PerfMonTile com.franco.kernel.quicktiles.PerfProfileTile com.franco.kernel.quicktiles.Script1Tile com.franco.kernel.quicktiles.Script2Tile com.franco.kernel.quicktiles.Script3Tile com.franco.kernel.services.AmbientLightSensorService com.franco.kernel.services.CpuTempService com.franco.kernel.services.FlashNotifService com.franco.kernel.services.ForegroundAppDetectionService com.franco.kernel.services.NewBatteryMonitorService com.franco.kernel.services.PerfMon com.franco.kernel.services.nightshift.DisableNightShiftService com.franco.kernel.services.nightshift.NightShiftService com.google.firebase.components.ComponentDiscoveryService}

**Comment by iBotPeaches** [comment]
Sorry, I have no idea what is being posted here or why.
